11-getting-started-with-Jupyter.Rmd

```diff
@@ -0,0 +1,306 @@
+
+## Chapter learning objectives
+
+By the end of the chapter, students will be able to:
+
+- use Jupyter to run, edit and write R code
+- open and view plain text data files in Jupyter
+- create new Jupyter notebooks
+- export Jupyter notebooks to other standard file types (e.g., `.html`, `.pdf`)
+
+## Jupyter
+
+Jupyter is a web-based interactive development environment for creating, editing and executing documents called Jupyter notebooks. Jupyter notebooks are documents that contain a mix of computer code (and its output) and formattable text. Given that they combine these two analysis artifacts in a single document—code is not separate from the output or written report—notebooks are one of the leading tools to create reproducible data analyses. Reproducible data analysis is one where you can reliably and easily recreate the same results when analyzing the same data. Although this sounds like something that should always be true of any data analysis, in reality, this is not often the case; one needs to make a conscious effort to perform data analysis in a reproducible manner.
+
+The name Jupyter came from combining the names of the three programming languages that it was initially targeted for (Julia, Python, and R), and now many other languages can be used with Jupyter notebooks.
+
+An example of what a Jupyter notebook looks like is shown in \@ref(fig:img-jupyter).
+
+```{r img-jupyter, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "A screenshot of a Jupyter Notebook.", fig.retina = 2}
+knitr::include_graphics("img/jupyter.png")
+```
+
+### Accessing Jupyter
+
+One of the easiest ways to start computing in R with Jupyter is to use a 
+web-based platform called JupyterHub that already has Jupyter, R, a number of R packages, and collaboration tools installed, configured and ready to use. 
+JupyterHub's are usually created and provisioned by organizations (for example,
+a university, a company, *etcetera*) and require authentication to gain 
+access. <!--- Insert link to public JupyterHub here if we can get 
+access/permission (either 2i2c collaboration, or 
+https://notebooks.gesis.org/hub/home) --> Jupyter can also be installed on your
+own computer (desktop or laptop), and we provide examples of how to do this in
+chapter \@ref(move-to-your-own-machine).
+
+## Code cells
+
+The sections of a Jupyter notebook that contain code in the R programming 
+language are referred to as code cells. Running a code cell will execute all of 
+the code it contains, and the output (if any exists) will be displayed directly
+underneath the code that generated it. Outputs may include printed text or 
+numbers, data frames and data visualizations.
+
+```{r code-cell-not-run, echo = FALSE, fig.cap = "A code cell in Jupyter that has not yet been executed. Note the absence of a number inside the square brackets to the left of the cell.", fig.retina = 2, out.width="100%"}
```

Comment:
```suggestion
```{r code-cell-not-run, echo = FALSE, fig.cap = "A code cell in Jupyter that has not yet been executed.", fig.retina = 2, out.width="100%"}
```